internal/coreconfig/coreconfig.go

```diff
@@ -359,6 +361,7 @@ func setDefaults() {
 	viper.SetDefault(string(CacheOperationsTTL), "5m")
 	viper.SetDefault(string(HistogramsMaxChartRows), 100)
 	viper.SetDefault(string(DebugPort), -1)
+	viper.SetDefault(string(DebugAddress), "")
```

Comment:
The suggest the defaults stays as `localhost`, as exposing the debug server externally is not something I believe users would want to do outside of a dockerized/k8s environment where they have carefully considered the security implications.

A follow-up PR with `firefly-cli` to have a flag to allocate ports to the debug ports of various services (`ethconnect`/`evmconnect`/`core` etc.) might be valuable for dev environments.

Reply:
Yeah I almost left the default as localhost - I'll revert it back. Happy to look at the firefly-cli/firefly-* PRs after this one.